### PR TITLE
rename frict model parameters to include model name [com1DFA]

### PR DESCRIPTION
**Pull request by awirb**
add check of frictModel and set other frictModel parameters to nan

add pytest, add checking for nan string in simDF to numerics

### CODE REVIEW

File: avaframe/com1DFA/checkCfg.py [hunk 1]
```diff
@@ -62,3 +63,49 @@ def checkCellSizeKernelRadius(cfg):
         log.info('sphKernelRadius is set to match meshCellSize of %s meters' % meshCellSize)
 
     return cfg
+
+
+def checkCfgFrictionModel(cfg):
+    """ check which friction model is chosen and set remaining friction model
+        parameters to ''
```
Comment (fso42) [inline avaframe/com1DFA/checkCfg.py:70]:
```suggestion
        parameters to nan
```

File: avaframe/com1DFA/com1DFA.py [hunk 2]
```diff
@@ -695,11 +695,33 @@ def createReportDict(avaDir, logName, relName, inputSimLines, cfg, reportAreaInf
...
+        reportST['Friction model'] = {'type': 'columns', 'model': 'Voellmy', 'mu': cfgGen['muvoellmy'], 'xsi': cfgGen['xsivoellmy']}
+        # reportST['Mu'] = cfgGen['muVoellmy']
+        # reportST['xsi'] = cfgGen['xsiVoellmy']
+    elif cfgGen['frictModel'].lower() == 'coulomb':
+        reportST['Friction model'] = {'type': 'columns', 'model': 'Coulomb', 'mu': cfgGen['mucoulomb']}
+        # reportST['Mu'] = cfgGen['muCoulomb']
+    elif cfgGen['frictModel'].lower() == 'wetsnow':
+        reportST['Friction model'] = {'type': 'columns', 'model': 'wetsnow', 'mu': cfgGen['mu0wetsnow'], 'xsi':  cfgGen['mu0wetsnow']}
+        # reportST['Mu'] = cfgGen['mu0WetSnow']
+        # reportST['xsi'] = cfgGen['xsiWetSnow']
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:723]: What are the comments for? If they are not needed anymore, please remove them

File: avaframe/com1DFA/com1DFA.py [hunk 3]
```diff
@@ -1523,6 +1545,7 @@ def DFAIterate(cfg, particles, fields, dem, inputSimLines, simHash=''):
     frictType = frictModelsList.index(frictModel) + 1
     log.debug('Friction Model used: %s, %s' % (frictModelsList[frictType-1], frictType))
 
+    print('frictModel is ', frictModel, frictType)
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:1548]: Seems like a debug print? Either log.debug or remove...

File: avaframe/in3Utils/cfgUtils.py [hunk 4]
```diff
@@ -630,6 +635,35 @@ def convertDF2numerics(simDF):
...
+        name: str
+            name of pandas dataframe column
+
+        Returns
+        --------
+        simDF: pandas dataframe
+            updated pandas dataframe with np.nan values where string nan was
+    """
+
+    nanIndex = simDFTest.str.match('nan')
```
Comment (fso42) [inline avaframe/in3Utils/cfgUtils.py:657]: Should we lower or set the IGNORECASE?
https://pandas.pydata.org/docs/reference/api/pandas.Series.str.match.html

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/in3Utils/cfgUtils.py (hunk 4 above)
Comment (awirb) [reply avaframe/in3Utils/cfgUtils.py]: I added flag re.IGNORECASE

**Review by fso42 — APPROVED** [review]

**Pull request merged**